src/QtLocationPlugin/QGCCachedTileSet.cpp

```diff
@@ -82,11 +87,17 @@ void QGCCachedTileSet::createDownloadTask()
 
 void QGCCachedTileSet::resumeDownloadTask()
 {
```

Comment:
Should we only resume if it is already canceled? If you click Ok or Cancel the UI stops updating the download and this value can get out of sync with whether the button shows to resume/cancel

Reply:
Okay since its merged now I guess it is not necessary anymore

Reply:
I tried it out and it didn't help the problem I was noticing, so I figured just merge this and I'll look more into it later. I think it's partly on the QML side of things

Reply:
Okay got it, thanks for explanation